checkall.sh

```diff
@@ -110,8 +110,10 @@ OUTPUT+="$PAYLOAD_FLAWFINDER"
 OUTPUT+=$'\n```\n' 
 
 PAYLOAD=$(echo '{}' | jq --arg body "$OUTPUT" '.body = $body')
+if [ $FLAWFINDER_LINE_NUMBER -gt 27 ]
+then
```

Comment:
can we rely on those line numbers?

Reply:
It's a bit of an ugly way, but that's the quickest solution I could think of, but if you have a better way, please let me know cause I hate hardcoding values like that!

Reply:
So far it 's been consistent from what I could gather

Reply:
no problem, if it's working well, that should be ok for now 👍